src/openforms/submissions/api/validation.py

```diff
@@ -17,6 +17,13 @@
 from ..models import Submission, SubmissionStep
 
 
+def privacy_policy_accepted(value: bool) -> None:
```

Comment:
```suggestion
def check_privacy_policy_accepted(value: bool) -> None:
```
maybe use `check` or `validate` as prefix?

Reply:
Mhmm indeed this would be nice! Although `CompletionValidationSerializer` also uses `privacy_policy_accepted`, so maybe I will leave it as it is to keep them consistent?

Reply:
isn't/was this part of this refactor/reorganization? I thought it was a serializer method only before and was now extracted into a re-usable validator function.